client/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CoursesComponent } from './components/courses/courses.component';
+import { ErrorPageComponent } from './components/error-page/error-page.component';
+import { CourseComponent } from './components/lesson/course.component';
+import { PageNotFoundComponent } from './components/page-not-found/page-not-found.component';
+import { appPaths } from './configs/app-paths';
+
+const routes: Routes = [
+  { path: appPaths['courses'], component: CoursesComponent },
```

Comment:
```suggestion
  { path: appPaths.courses, component: CoursesComponent },
```
Why do you get the property from the object this way?

Reply:
That's default Angular Linter rule, it made me imperatively  write by this exact way.

Reply:
Thanks for the clarification.